code/dynamic_programming/src/matrix_chain_multiplication/matrix_chain_multiplication.cpp

```diff
@@ -1,7 +1,12 @@
 #include <climits>
 #include <cstdio>
+#include <iostream>
```

Comment:
Separate into sources and test.

Reply:
Sir, Can you help me out with this?
Thank you.

Reply:
[The src](https://github.com/OpenGenus/cosmos/blob/master/code/data_structures/src/tree/binary_tree/binary_tree/diameter/diameter.cpp) and [the test](https://github.com/OpenGenus/cosmos/blob/master/code/data_structures/test/tree/binary_tree/binary_tree/diameter/test_diameter.cpp) of diameter tree is an example.